Documentation/Books/Manual/Administration/Configuration/SSL.md

```diff
@@ -74,11 +74,11 @@ the certificate.
 
 `--ssl.cafile filename`
 
```

Comment:
its not whenever the client sends requests; its whenever the client establishes an ssl connection.